introduction/es/03-how-works.asciidoc

```diff
@@ -0,0 +1,41 @@
+== ¿Cómo funciona InnerSource?
+
+Pongamos como ejemplo la situación donde un equipo A usa el software que produce un equipo B.
+El equipo A tiene una petición para una nueva funcionalidad y se la envía al equipo B, pero el equipo B no llega a tiempo de implementar dicha mejora para el equipo A.
+En un entorno de InnerSource, si al equipo A no le facilitan dicha funcionalidad entonces habría enviado un pull request en su lugar.
+Esto significa que el equipo A implementa la funcionalidad directamente en el software del equipo B y envía un pull request para que sea revisado con los cambios pertinentes.
+El equipo B es entonces el encargado de revisar dicho código y aceptarlo cuando esté listo.
+
+En este ejemplo, el equipo A es el _Invitado_ y el equipo B es el equipo _Anfitrión_.
+Los términos de _Invitado_ y _Anfitrión_ se usan para referirse a una situación análoga a la de tener una visita en casa.
+En esta situación, la mayor parte de las personas quieren ser un buen anfitrión y para ello hay que asegurar que la casa está ordenada y las cosas guardadas en su sitio como anticipo a la visita de nuestros invitados. Los visitantes serán entonces bienvenidos en la puerta e invitados a pasar y podrán usar las diferentes herramientas e instalaciones que sean públicas.
+Hay, por supuesto, una serie de relas en la casa que se pide a los invitados que sigan.
+De igual manera, los invitados quieren ser educados y mostrar respeto por la casa y por los anfitriones. Tienen cuidado con las cosas de la casa y siguen las reglas durante su estancia. Además esperan que quizá vuelvan a ser invitados de nuevo siempre y cuando hayan sido educados y corteses.
+Estos conceptos acerca de una visita en una casa son una metáfora de la actitud  y comportamientos que los equipos deben tener cuando actúan como anfitriones o como invitados a la hora de contribuir al código fuente de nuestro producto.
+
+Echemos un vistazo más en detalle del proceso de InnerSource.
+Para ilustrar esta explicación usaremos unos pocos roles que son clave en los equipos que actúan como invitados y anfitriones.
+En primer lugar, el https://innersourcecommons.org/resources/learningpath/product-owner/index[_Product Owner_] determina qué funcionalidad puede ser aceptada por el equipo anfitrion como una contribución.
```

Comment:
Does the term _Product Owner_ have a Spanish translation in the Agile world?

Reply:
Not in reality (at last as far as I know!). We use product owner or PO.

I'd say this works in Spain (not sure about how this is translated, if so, in other Spanish-speaking countries).